test/base/test_collector.py

```diff
@@ -60,16 +70,16 @@ def test_collector():
     policy = MyPolicy()
     env = env_fns[0]()
     c0 = Collector(policy, env, ReplayBuffer(size=100, ignore_obs_next=False),
-                   preprocess_fn)
-    c0.collect(n_step=3, log_fn=logger.log)
+                   logger.preprocess_fn)
+    c0.collect(n_step=3)
     assert np.allclose(c0.buffer.obs[:3], [0, 1, 0])
     assert np.allclose(c0.buffer[:3].obs_next, [1, 2, 1])
-    c0.collect(n_episode=3, log_fn=logger.log)
+    c0.collect(n_episode=3)
     assert np.allclose(c0.buffer.obs[:8], [0, 1, 0, 1, 0, 1, 0, 1])
     assert np.allclose(c0.buffer[:8].obs_next, [1, 2, 1, 2, 1, 2, 1, 2])
     c0.collect(n_step=3, random=True)
```

Comment:
It should be changed since single-env does not exactly collect n_episode and n_step. Check for other cases nearby.